exorelay-javascript/features/support/steps-given.ls

```diff
@@ -16,11 +16,11 @@ module.exports = ->
 
 
   @Given /^a new ExoRelay instance connecting to port (\d+)$/ (@exocom-port) ->
-    @exo-relay = new ExoRelay {exocom-host: 'localhost', @exocom-port, role: @role = 'test-service'}
+    @exo-relay = new ExoRelay {exocom-host: 'localhost', @exocom-port, role: @role = 'test-service', service-type: @role = 'test-service'}
```

Comment:
You assigning content to `@role` while providing it as an argument to the ExoRelay constructor. These are two separate activities that should happen on separate lines.